### PR TITLE
Allow ssl params to be passed

### PR DESCRIPTION
**Pull request by tonyliu-minted**
Jira:

http://jira.minted.com/browse/CM-11500

Test:

Verified by hacking this code in vagrant box and using ssl to communicate with
test server

screen shot 2016-09-12 at 2 50 18 pm

### CODE REVIEW

**Comment by gabriella-minted** [comment]
🚢

**Comment by gabriella-minted** [comment]
If you update the version, please update the setup.py file for this repo too; maybe `0.5.2.1` or something

**Comment by gabriella-minted** [comment]
Next step: please update message service to use this repo. Similar change that I made: https://github.com/minted/message_service/pull/388/files

File: client.py [hunk 1]
```diff
@@ -90,12 +90,17 @@ def DEBUG(self,text,severity='info'):
 import transports,dispatcher,auth,roster
 class CommonClient:
     """ Base for Client and Component classes."""
-    def __init__(self,server,port=5222,debug=['always', 'nodebuilder']):
+    def __init__(self,server,port=5222,debug=['always', 'nodebuilder'],
+        cert_reqs=None, ca_certs=None):
```
Comment (gabriella-minted) [inline client.py]: ⛔  has to be one of the enum values for cert_reqs

Comment (tonyliu-minted) [reply client.py]: addressed

**Pull request merged**